src/javaRosa.js

```diff
@@ -792,7 +792,12 @@ define([
     };
 
     var itextMediaBlock = function (mug, options) {
-        var block = itextConfigurableBlock(mug, options);
+        var block = itextConfigurableBlock(mug, options),
+            pathPrefix = options.pathPrefix;
+
+        if(!_.isString(options.pathPrefix)) {
```

Comment:
Trying to pin down why this is problematic. A non-string value other than `null` or `undefined` is probably a bug, and using `_.isString` means it will be silently discarded/overwritten. The alternate check for `null` and/or `undefined` is more strict; other (likely buggy) non-string values will pass on through to a place where they will either cause an error (best outcome) or hopefully be displayed in a place where a user can see it and report a bug.